modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml

```diff
@@ -70,7 +70,8 @@
 				<version>@com.liferay.css.builder.version@</version>
 				<executions>
 					<execution>
-						<phase>generate-sources</phase>
+						<id>build-css</id>
```

Comment:
I'm sorry @lawrence-lee, you haven't told me why you need these IDs 😋 Does the Maven build fail without them?